lib/Backend/LowerMDShared.cpp

```diff
@@ -8497,17 +8497,22 @@ LowererMD::LowerReinterpretPrimitive(IR::Instr* instr)
 #elif _M_IX86
```

Comment:
Yeah, this was initially for wasm `reinterpret` operator.
It converts an int64 to double or double to int64 or int32 to float or float to int32, so I'm not sure exactly how I would name it.